package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "dragon-ball-api",
+  "version": "1.0.0",
+  "description": "You will be building a working API and front-end. This is part 1 of 2. Create a new repo in GitHub for your project, make your instructor a collaborator, and clone it locally. Remember to work in and part-one-answer branch, not master In this part of your project you will be building the front-end for your API site. The design and content of your front-end is up to you, the only requirement is that it provide documentation for using the API you will be building Your first progress check with your instructor will be the week of April 19th You should push all progress to GitHub prior to your meeting with the instructor Your graded submission for this part will be due on Thursday April 30th before class",
+  "main": "index.js",
+  "scripts": {
+    "dev": "./node_modules/.bin/concurrently \"npm run dev:client\" \"npm run dev:server\"",
+    "dev:client": "./node_modules/.bin/env-cmd ./node_modules/.bin/webpack --porgress --inline --config webpack.dev.js",
+    "dev:server": "./node_modules/.bin/env-cmd ./node_modules/.bin/babel-watch server.js",
+    "dev:migrate": "./node_modules/.bin/env-cmd ./node_modules/.bin/sequelize db:migrate --env development --config ./configs/sequelize.js",
+    "lint": "./node_modules/.bin/eslint --format codeframe .",
+    "lint:fix": "./node_modules/.bin/eslint --fix --format codeframe .",
+    "test": "./node_modules/.bin/env-cmd ./node_modules/.bin/mocha --require @babel/register --require ./tests/setup.js ./tests/**/*.test.js*"
```

Comment:
So the point of .env-sample is so I create my .env correctly. When you leave off things, this happens.

![image](https://user-images.githubusercontent.com/5775716/85063481-3618ac80-b178-11ea-8366-131f2bc3acbb.png)

But when I fix it, your tests are mostly good so I suppose we can give you some of the extra credit.

![image](https://user-images.githubusercontent.com/5775716/85063663-7841ee00-b178-11ea-92a6-35046b7b250a.png)